src/module_2/EDA_1.md

```diff
@@ -0,0 +1,3314 @@
+    Error checking duplicates in Inventory: unhashable type: 'numpy.ndarray'
+    Null values in Inventory: Series([], dtype: int64)
+    
+    Duplicates in Users: 0
+    Null values in Users: user_nuts1          51
+    count_people      4658
+    count_adults      4658
+    count_children    4658
+    count_babies      4658
+    count_pets        4658
+    dtype: int64
+
+
+
+    
+![png](EDA_1_files/EDA_1_0_1.png)
+    
+
+
+
+    ---------------------------------------------------------------------------
+
+    ValueError                                Traceback (most recent call last)
+
+    Cell In[27], line 102
+         99 axes[0].set_title('Relationship between Price and Compare At Price')
+        101 # Correlation matrix
+    --> 102 correlation_matrix = orders.corr()
+        103 sns.heatmap(correlation_matrix, annot=True, fmt=".2f", ax=axes[1])
+        104 axes[1].set_title('Correlation Matrix of Orders')
+
+
+    File ~/.cache/pypoetry/virtualenvs/zrive-ds-O-AZ-S6U-py3.11/lib/python3.11/site-packages/pandas/core/frame.py:10054, in DataFrame.corr(self, method, min_periods, numeric_only)
+      10052 cols = data.columns
+      10053 idx = cols.copy()
+    > 10054 mat = data.to_numpy(dtype=float, na_value=np.nan, copy=False)
+      10056 if method == "pearson":
+      10057     correl = libalgos.nancorr(mat, minp=min_periods)
+
+
+    File ~/.cache/pypoetry/virtualenvs/zrive-ds-O-AZ-S6U-py3.11/lib/python3.11/site-packages/pandas/core/frame.py:1838, in DataFrame.to_numpy(self, dtype, copy, na_value)
+       1836 if dtype is not None:
+       1837     dtype = np.dtype(dtype)
+    -> 1838 result = self._mgr.as_array(dtype=dtype, copy=copy, na_value=na_value)
+       1839 if result.dtype is not dtype:
+       1840     result = np.array(result, dtype=dtype, copy=False)
+
+
+    File ~/.cache/pypoetry/virtualenvs/zrive-ds-O-AZ-S6U-py3.11/lib/python3.11/site-packages/pandas/core/internals/managers.py:1732, in BlockManager.as_array(self, dtype, copy, na_value)
+       1730         arr.flags.writeable = False
+       1731 else:
+    -> 1732     arr = self._interleave(dtype=dtype, na_value=na_value)
+       1733     # The underlying data was copied within _interleave, so no need
+       1734     # to further copy if copy=True or setting na_value
+       1736 if na_value is not lib.no_default:
+
+
+    File ~/.cache/pypoetry/virtualenvs/zrive-ds-O-AZ-S6U-py3.11/lib/python3.11/site-packages/pandas/core/internals/managers.py:1794, in BlockManager._interleave(self, dtype, na_value)
+       1792     else:
+       1793         arr = blk.get_values(dtype)
+    -> 1794     result[rl.indexer] = arr
+       1795     itemmask[rl.indexer] = 1
+       1797 if not itemmask.all():
+
+
+    ValueError: could not convert string to float: '62e271062eb827e411bd73941178d29b022f5f2de9d37f50190a1d6741e02be71414684ca6df992c186522a0433f10367d18d7102da301989cb7929559747eda'
+
+
+
+    
+![png](EDA_1_files/EDA_1_0_3.png)
+    
+
+
```

Comment:
Each cell must have a goal. After executing the cell I expect a comment on what is the conclusion you get of that particular exploration.